webapp/nuxt.config.js

```diff
@@ -21,10 +21,10 @@ module.exports = {
 
   modern: !dev ? 'server' : false,
 
-  transition: {
-    name: 'slide-up',
-    mode: 'out-in',
-  },
+  // transition: {
+  //  name: 'slide-up',
+  //  mode: 'out-in',
+  // },
```

Comment:
is there a reason we aren't either deleting this, or changing it to `pageTransition`?

Reply:
My motivation to disable, not remove it: I could not see any difference in removing or adding it. However it is a documented feature. Maybe w'ere doing something wrong? Maybe I'm blind?

I can also remove it, if that helps you @mattwr18

Reply:
well, I don't think it would help me so much personally, hehe, the reason I ask is because it can be easy to collect comments in the code base and if we don't want/need some code, let's delete it, I think.

I pushed up some changes, if you can review. 

I had a look at the docs, and it seems like this config would be used anytime we have a `slide-up` transition, which we have a handful of, so I think it might be best to leave it, not remove it.
https://nuxtjs.org/api/pages-transition/
https://vuejs.org/v2/guide/transitions.html#Transition-Modes